src/components/Footer/index.js

```diff
@@ -1,15 +1,270 @@
+              <Link
+                to={
+                  item.id
+                    ? interpolatePath(PATHS.PATHWAY_COURSE, {
+                        pathwayId: item.id,
+                      })
+                    : item.link
+                }
+                style={{
+                  textDecoration: "none",
+                }}
+              >
+                <Typography
+                  variant="body2"
+                  color="text.primary"
+                  sx={{ pb: "5px" }}
+                >
+                  {item.title}
+                </Typography>
+              </Link>
+            );
+          } else {
+            return (
+              <a
+                style={{
+                  textDecoration: "none",
+                }}
+                target="_blank"
+                href={item.link}
+              >
+                <Typography
+                  variant="body2"
+                  color="text.primary"
+                  sx={{ pb: "5px" }}
+                >
+                  {item.title}
+                </Typography>
+              </a>
+            );
+          }
+        })}
+      </List>
+    </>
+  );
+};
+
+function FooterIcon(props) {
+  const classes = useStyles();
+  return (
+    <Box className={classes.image}>
+      <img
+        src={require("./asset/" + props.name + ".svg")}
+        alt={props.alt || "image of " + props.name}
+        loading="lazy"
+      />
+    </Box>
+  );
+}
 
 function Footer() {
+  const classes = useStyles();
+  const [showHeader, setShowHeader] = React.useState(true);
+  const dispatch = useDispatch();
+  const { data } = useSelector((state) => state.Pathways);
+
+  useEffect(() => {
+    dispatch(pathwayActions.getPathways());
+  }, [dispatch]);
+
+  data &&
+    data.pathways.forEach((pathway) => {
+      menu.LearningTracks.forEach((item) => {
+        if (pathway.code === item.code) {
+          item["id"] = pathway.id;
+        }
+      });
+    });
+
+  useEffect(() => {
+    if (
+      window.location.pathname.split("/").includes("course-content") ||
+      window.location.pathname.split("/").includes("login") ||
+      window.location.pathname.split("/").includes("profile")
+    ) {
+      console.log("here");
+      setShowHeader(false);
+    }
+  }, []);
+
+  const history = useHistory();
+  history.listen((location, action) => {
+    if (
+      location.pathname.split("/").includes("course-content") ||
+      location.pathname.split("/").includes("login") ||
+      location.pathname.split("/").includes("profile")
+    ) {
+      console.log("not in header");
+      setShowHeader(false);
+    } else {
+      setShowHeader(true);
+    }
+  });
+
   return (
-    <div className="footer">
-      <Link className="footer-link" to={PATHS.PRIVACY_POLICY}>
-        Privacy Policy
-      </Link>
-    </div>
+    <Box
+      style={{
+        display: showHeader ? "inherit" : "none",
+      }}
+      maxWidth="false"
+      bgcolor="primary.light"
```

Comment:
Use the following so that the appropriate `<footer>` element is used when the HTML is generated.
```suggestion
      bgcolor="primary.light"
      component="footer"
```